src/main/java/com/jcabi/github/RtPull.java

```diff
@@ -159,6 +161,42 @@ public void merge(
     }
 
     @Override
+    public MergeState merge(
+        @NotNull(message = "message can't be NULL") final String msg,
+        final String sha) throws IOException {
+        final JsonObjectBuilder builder = Json.createObjectBuilder()
+            .add("commit_message", msg);
+        if (sha != null) {
```

Comment:
@sonOfRa I think we should not allow to null sha in this method.
If sha is not provided then another method ```merge``` will be used (without sha)